src/data/finset/lattice.lean

```diff
@@ -1112,6 +1112,10 @@ lemma sup_eq_bUnion {α β} [decidable_eq β] (s : finset α) (t : α → finset
   s.sup t = s.bUnion t :=
 by { ext, rw [mem_sup, mem_bUnion], }
 
+lemma sup_id_eq_sUnion {α} (s : finset (set α)) :
+  s.sup id = ⋃₀(↑s) :=
+sup_id_eq_Sup _
```

Comment:
Can you add the version of this for `inf` / `sInter` too?

Reply:
Why did you mark one as @[simp] and not the other one?
Was it intended ?
I moved the sup/sUnion lemmas after `sup_eq_supr`, and wrote the inf/sInter ones just after `inf_eq_infi`.

Reply:
Nope, it was accidental!

Reply:
> I moved the sup/sUnion lemmas after `sup_eq_supr`, and wrote the inf/sInter ones just after `inf_eq_infi`.

Looks like you still need to remove this one

Reply:
Oops…